kubeshop/monokle
# feat: edit toggle in cluster mode

This PR...

## Changes
Add new Settings option: Allow editing resources

## How to test it
- 1 Go to Setting -> disable "Allow editing resources" under "Cluster Mode"
- 2 Enable "Cluster Mode"
- 3 Should not be able to edit resources

## Screenshots
![image](https://user-images.githubusercontent.com/12400110/164514011-25819457-0819-40eb-bfda-ff5fcc837877.png)

## Checklist
